src/vm/database/sqlite.rs

```diff
@@ -121,22 +121,22 @@ impl ContractDatabaseConnection {
         let sql = "SELECT sql FROM sqlite_master WHERE name=?";
         let _: String = self.conn.query_row(sql, &["maps_table"],
                                             |row| row.get(0))
-            .map_err(|x| Error::new(ErrType::SqliteError(IncomparableError{ err: x })))?;
+            .map_err(|x| RuntimeErrorType::SqliteError(IncomparableError{ err: x }))?;
```

Comment:
It is possible to recover from this error?

Reply:
Probably not -- but in the interest of a friendlier CLI interface, we'd want this to be a catchable error. This should be an InterpreterError, I think, instead of a Runtime. Will update.

Reply:
Changed to InterpreterError